### PR TITLE
acrn-config: fix the issue config app failed to generate patch for a new board

### PR DESCRIPTION
**Pull request by szhen11**
1. acrn-config: change source file format to unix
2. acrn-config: fix the issue config app failed to generate patch for a new board

Tracked-On: #3760
Signed-off-by: Shuang Zheng <shuang.zheng@intel.com>
Reviewed-by: Victor Sun <victor.sun@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: misc/acrn-config/config_app/controller.py [hunk 1]
```diff
@@ -1,288 +1,291 @@
...
-            if not element.tail or not element.tail.strip():
-                element.tail = i
-            for element in element:
-                self._format_xml(element, depth + 1)
-            if not element.tail or not element.tail.strip():
-                element.tail = i
-        else:
-            if depth and (not element.tail or not element.tail.strip()):
-                element.tail = i
+# Copyright (C) 2019 Intel Corporation.
```
Comment (binbinwu1) [inline misc/acrn-config/config_app/controller.py]: Why there are diffs for the parts that are not modifed?
It is difficult to review the modified part...

Comment (szhen11) [reply misc/acrn-config/config_app/controller.py]: @binbinwu1 there are 2 patches for this PR. The first one is: acrn-config: change source file format from dos(line ending as CRLF) to unix format(line ending as LF). You can find the diffs from maillist.

File: misc/acrn-config/config_app/views.py [hunk 2]
```diff
@@ -452,7 +452,13 @@ def upload_board_info():
                 for generic_name in os.listdir(os.path.join(config_path, 'generic')):
                     generic_file = os.path.join(config_path, 'generic', generic_name)
                     if os.path.isfile(generic_file):
-                        copyfile(generic_file, os.path.join(config_path, board_type, generic_name))
+                        new_file = os.path.join(config_path, board_type, generic_name)
+                        copyfile(generic_file, new_file)
+                        xml_config = XmlConfig(os.path.join(current_app.config.get('CONFIG_PATH'),
+                                                            board_type))
+                        xml_config.set_curr(generic_name[:-4])
```
Comment (binbinwu1) [inline misc/acrn-config/config_app/views.py]: what's the last 4 elements to skip?

Comment (szhen11) [reply misc/acrn-config/config_app/views.py]: The generic_name is the configuration xml name like "industry.xml". xml_config.set_curr() shall accept the file name without suffix name as the parameter, so need to skip the last 4 characters.

**Pull request merged**